pkg/cloudprovider/providers/azure/azure_standard_test.go

```diff
@@ -90,3 +90,38 @@ func TestGetLastSegment(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateStorageAccountName(t *testing.T) {
+	tests := []struct {
+		prefix string
+	}{
+		{
+			prefix: "",
+		},
+		{
+			prefix: "pvc",
+		},
+		{
+			prefix: "1234512345123451234512345",
+		},
+	}
+
+	for _, test := range tests {
+		accountName := generateStorageAccountName(test.prefix)
+		if len(accountName) > storageAccountNameMaxLength || len(accountName) < 3 {
+			t.Errorf("input prefix: %s, output account name: %s, length not in [3,%d]", test.prefix, accountName, storageAccountNameMaxLength)
+		}
+
+		valid := true
+		for _, char := range accountName {
+			if (char < 'a' || char > 'z') && (char < '0' || char > '9') {
+				valid = false
```

Comment:
can just call t.Errorf here, so that Errorf it can show which char is out of range.

Reply:
fixed